pootle/apps/pootle_translationproject/models.py

```diff
@@ -221,7 +221,7 @@ def checker(self):
             checkerclasses = [import_func(settings.POOTLE_QUALITY_CHECKER)]
         else:
             checkerclasses = [
-                checks.projectcheckers.get(self.project.checkstyle,
+                PROJECT_CHECKERS.get(self.project.checkstyle,
```

Comment:
I don't think this will work, because the customer checker you can set in settings now won't be available.